lib/main.dart

```diff
@@ -73,7 +73,9 @@ class _StartScreenState extends State<StartScreen> {
   @override
   void didChangeDependencies() {
     super.didChangeDependencies();
-    _optionsByType = _contentProvider.findAllOptionsByType();
+    _content = _contentProvider.findAllOptionsByType().then(
```

Comment:
Nit: We might want to check for the content options via getters first, e.g. `presetPronounOptions`. If we only need the Pronoun options or if all of the options are already loaded, this isn't ideal. 

I think all the options are something like 100-200kb of uncompressed data (in json format) at the moment, so this is not a big deal.

Reply:
This query is triggered once per app start. I figured it would be cleaner to have query done once and then elsewhere we can assume the data is there, otherwise we need to put checks and queries everywhere else we plan to use these values. But yes maybe we can look into how we can optimize this further.